en/health/centreon-agent.md

```diff
@@ -0,0 +1,511 @@
+
+    ```yaml
+    gateway:
+      enable: true
+      listen_port: [listening-port]
+      auth_token: [your-gateway-token]
+    ```
+    
+    Example:
+
+    ```yaml
+    gateway:
+      enable: true
+      listen_port: 54321
+      auth_token: azerty1234
+    ```
+
+    You then need to restart the Agent
+
+    ```
+    systemctl restart centreon-agent.service
+    ```
+
+- Gateway Client
+
+    In a Gateway configuration, the Gateway Client delegates the configuration of its main token to the Gateway Server (since only the latter communicates with our platform).
+    As a consequence, the `token` line needs to be commented with the yaml comment operator “#”.
+    If you have defined an authentication token (`auth_token`) on the gateway server, you need to add it to the configuration of the gateway client too. 
+
+    ```yaml
+    output:
+    #token: [your-token]
+      gateway:
+        url: http://[gateway-server-ip-address]:[listening-port]
+        auth_token: [your-gateway-token]
+    ```
+
+    Example:
+
+    ```yaml
+    output:
+    #token: aaaa-aaaa-aaaa-aaaa
+      gateway:
+        url: http://172.28.6.145:54321
+        auth_token: azerty1234
+    ```
+
+    You then need to restart the Agent
+
+    ```
+    systemctl restart centreon-agent.service
+    ```
+
+### Enabling the Collection of Centreon Logs
+
+Starting from version 2 and up of the **centreon-agent**, logs generated by the monitored Centreon component can be collected. 
+
+To define which logs must be collected, you need to create yml configuration files in the following folder: `/etc/centreon-agent/conf.d`.
+To collect a specific log, the configuration file must contain the following arguments: path, type and pattern of the target log file. 
+You can have several configuration files - each file is parsed and its target log files are added to the collection. 
+
+#### Using the Templates
+
+To make log collection configuration easier, pre-configured templates are provided. Each template covers a specific scope depending on the target Centreon component, its version, etc.
+
+Templates are located in the following folder:
+
+```
+/usr/share/centreon-agent/examples
+```
+
+Based on your monitored Centreon component you can simply copy/paste the corresponding template to your `/etc/centreon-agent/conf.d` folder.
+
+#### Finalize Templates Configuration
+
+>For a Centreon Poller, log files are prefixed with the Poller’s name so you need to adapt the Poller template:
+>Open the Poller template and replace all `<pollername>` placeholders within the “path” section with the actual Poller’s name.
```

Comment:
reading from the templates, the real placeholder is not  <pollername> but `POLLERNAME`